src/main/java/calculator/enums/MenuType.java

```diff
@@ -0,0 +1,27 @@
+package calculator.enums;
+
+public enum MenuType {
+
+  HISTORY("1"),
+  CALCULATE("2"),
+  //메뉴에 없는 숫자 입력시 EXIT 상태가 되는데, EXIT의 다른 값을 -1로 설정해놓았습니다.
+  //enum에 정의된 EXIT 상태와 실제 종료 조건이 다른 것을 고치고 싶습니다.
+  EXIT("-1");
+
+  final private String value;
+  MenuType(String value) {
+    this.value = value;
+  }
+
```

Comment:
```suggestion
  public MenuType findByMenu(String op) {
	  return Arrays.stream(values())
		  .filter(menu -> menu.getValue().equals(op))
		  .findFirst()
		  .orElse(EXIT);
  }
```

enum의 정의는 상수를 미리 정의한 열거 타입으로 볼 수 잇는데, 이를 활용하기 위해서 이러한 로직이 들어가면 조건문을 통해 로직처리를 해야할 일이 줄어들 것 같습니다.

Reply:
확실히 지금 저는 enum을 잘 다루지 못하는것 같습니다. 좋은 방법 알려주셔서 감사합니다!